.travis.yml

```diff
@@ -74,6 +77,8 @@ before_install:
       bash bin/install-wp-tests.sh wordpress_test root '' localhost $WP_VERSION
       set +e
     fi
+after_success:
```

Comment:
this is sketchy. we should probably add it to the repo (and review it) instead.

Reply:
I added it to my repo for Aperture because I didn't like the idea of running code from someone else's repo that might change at a moments notice. This is how the terrible npm hacks happen so I would like to avoid that.

Reply:
I think he's talking about the deployment script, but okay.

Reply:
(kidding! this one would be understandable, and not as painful as eg semantic linkbacks. i still vote for bundling though. 😁)

Reply:
After I merge the script, how do you feel about a release? Do we need more user feedback?

Reply:
i think we still have outstanding user reports of bugs and things not working, right? we should solicit more testers first, and then make sure all of those reports are fixed.

Reply:
I am waiting for information from some people

Reply:
But I want to be ready to go